nixio/property.py

```diff
@@ -145,6 +145,12 @@ def unit(self, new):
 
     @property
     def uncertainty(self):
+        dataset = self._h5dataset
+        x, y, z = dataset._parent.file.attrs["version"]
+        if y < 1 or z < 1:
```

Comment:
Best to check if the entire ver tuple is `< (1, 1, 1)`.

Reply:
resolved